Command and Control/Network communication disovery

```diff
@@ -0,0 +1,18 @@
+// Query 1 shows you any network communication happened from endpoints to an specific Remote IP or Remote URL
+// Ensure to update RemoteIP and RemoteURL variable.
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == "IP ADDRESS GOES HERE"
+or RemoteUrl contains "URL GOES HERE"
+| project EventTime, ComputerName, ActionType, RemoteIP, RemoteUrl, InitiatingProcessFileName, InitiatingProcessCommandLine
+
+// Query 2 shows you any network communication that happened from endpoints through a specific file to an Remote IP or Remote URL
+//Ensure to update RemoteIP, RemoteURL and InitatingProcessFileName
+// For questions @MiladMSFT on Twitter or milad.aslaner@microsoft.com by email
+NetworkCommunicationEvents
+| where EventTime > ago(30d)
+| where RemoteIP == " IP ADDRESS GOES HERE"
+or RemoteUrl contains " URL GOES HERE"
```

Comment:
should have parenthesis arround "or" before the next "and"

Reply:
Why do I need parenthesis in that situation? The query works without it.

Reply:
Sorry - maybe I put this comment on a wrong line in your query.
if you have "cond1 or cond2 and cond3", it'll actually run "cond1 or (cond2 and cond3)" due to operator precedence.
in this query you actually want to do (cond1 or cond2) and cond3 - so should explicitly use parenthesis.
(and in general, if you involve multiple different operators - e.g. and/or - so it's a best-practice to use parenthesis as it helps to avoid mistakes)